worker/src/RTC/RtpPacket.cpp

```diff
@@ -16,32 +16,16 @@ namespace RTC
 {
 	/* Class variables. */
 
-	thread_local uint32_t RtpPacket::nextMediasoupPacketId{ 0u };
+	thread_local uint32_t RtpPacket::nextMediasoupPacketId{ Utils::Crypto::GetRandomUInt(
+		1u, std::numeric_limits<uint32_t>::max() / 2) };
 
 	/* Class methods. */
 
 	uint32_t RtpPacket::GetNextMediasoupPacketId()
 	{
 		MS_TRACE();
 
-		// Make RtpPacket::nextMediasoupPacketId first value be random and then
-		// increase it by one, and don't let it be 0.
-		if (RtpPacket::nextMediasoupPacketId == 0)
-		{
-			RtpPacket::nextMediasoupPacketId =
-			  Utils::Crypto::GetRandomUInt(1u, std::numeric_limits<uint32_t>::max() / 2);
-		}
-		else
-		{
-			RtpPacket::nextMediasoupPacketId++;
-
-			if (RtpPacket::nextMediasoupPacketId == 0)
-			{
-				RtpPacket::nextMediasoupPacketId = 1;
-			}
-		}
-
-		return RtpPacket::nextMediasoupPacketId;
+		return RtpPacket::nextMediasoupPacketId++;
```

Comment:
But now it can be 0 which was not possible before

Reply:
Please let's not limit C++ benefits and optimisations with the excuse of the limitations in Rust. We want complex C++ code to prove how smart we are.

BTW ++ operator exists in every language.

Reply:
> But now it can be 0 which was not possible before

What's the problem with being 0 upon wrap up? Should we avoid 0 and not 1 or 2 for any rason?

Reply:
++ operator is clear and obvious in this case.